.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Proposed changes
+
+<!--
+Describe an overview of the changes, why they are useful, and any comments on the implementation so that it is easy for reviewers to understand the PR.
+-->
+
+### Types of changes:
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+
+### Component:
+
+- [ ] Code
+- [ ] Documentation
+- [ ] Build system
+- [ ] Continuous integration
+
+### Code review checklist
+
+- [ ] Follows [code review guidelines](https://sxs-collaboration.github.io/spectre/code_review_guide.html)
+- [ ] Correct exception specification
+- [ ] Mark objects `const` whenever possible
+- [ ] Almost always `auto`
+- [ ] Passes clang-tidy, CppCheck, and has no compiler warnings
+- [ ] All dependent changes have been merged
+- [ ] Code is tested
```

Comment:
Also that tests pass, and code coverage is sufficiently high. If some code is unreachable or currently unimplemented then //LCOV_EXCL should be used.